index.html

```diff
@@ -1,311 +1,321 @@
+                </div>
+                <div class="title experience">
+
+                    <ul class="flex-container-list">
+                        <li>
+                            <h2 class="mobile-version-h2">CANOPY</h2>
+                            <h2 class="desktop-version-h2">Uber</h2>
+                        </li>
+                        <li class="mobile-version-li">Back End Dev</li>
+                        <li class="desktop-version-li1">Lead Developer</li>
+                        <li class="mobile-version-li2">2015</li>
+                        <li class="desktop-version-li2">2018</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <p class="mobile-para">A daily selection of privately<br> personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <p class="desktop-para">A smart assistant to make driving more safe, efficient, and fun by unlocking
+                        your most expensive computer: your car.</p>
+
+                </div>
+                <div class="languages">
+                    <ul class="flex-lang">
+                        <li><span>html</span></li>
+                        <li><span class="desktop-version-span-container">Ruby on rails</span></li>
+                        <li><span>css</span></li>
+                        <li><span>javaScript</span></li>
+                    </ul>
+                </div>
+                <div class="btn-see-projects">
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+
+        </section>
+
+        <section class="about">
+            <div class="grid-container-desktop-about">
+                <div class="title-about">
+                    <h1>About<br>Myself</h1>
+                </div>
+
+                <div class="paragraph-about">
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through
+                        some of my work and experience! If you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.</p>
+                </div>
+
+                <div class="container-social-about">
+                    <a href="#" class="button-connect-about">LET’S CONNECT</a>
+                    <ul class="social-icons">
+                        <li><img src="/img/social/Twitter.svg" alt=""></li>
+                        <li><img src="/img/social/Linkedin.svg" alt=""></li>
+                        <li><img src="/img/social/Vector-M.svg" alt=""></li>
+                        <li><img src="/img/social/Github.svg" alt=""></li>
+                        <li><img src="/img/social/Hand2.svg" alt=""></li>
+                    </ul>
+                </div>
+
+                <div class="btn-see-projects-about">
+                    <a href="#" class="button-about">Get my resume</a>
```

Comment:
- [OPTIONAL] Please style your button for hover and active as in the Figma template. It's a project requirement. As not required by the previous reviewer I consider this optional.